cleanlab/segmentation/rank.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""
+Methods to rank and score images in a semantic segmentation dataset , based on how likely they are to contain label errors.
+
+
+"""
+
+import pandas as pd
+import numpy as np
+from typing import List, Optional, Union, Tuple
+
+from cleanlab.segmentation.filter import find_label_issues
+
+
+def get_label_quality_scores(
+    labels: np.ndarray,
+    pred_probs: np.ndarray,
+    method: str = "softmin",
+    batch_size: int = 10000,
+    n_jobs: Optional[int] = 1,
+    verbose: bool = True,
+    **kwargs
+) -> Tuple[np.ndarray,np.ndarray]:
```

Comment:
```suggestion
def get_label_quality_scores(
    labels: np.ndarray,
    pred_probs: np.ndarray,
    *,
    method: str = "softmin",
    batch_size: int = 10000,
    n_jobs: Optional[int] = 1,
    verbose: bool = True,
    **kwargs
) -> Tuple[np.ndarray,np.ndarray]:
```

Make optional args kwargs